content/en/articles/js/flow/index.md

```diff
@@ -0,0 +1,290 @@
+---
+date: "2016-06-07"
+title: Introduction to Flow, to make your code run on the first try
+tags:
+  - javascript
+  - flow
+  - type
+authors:
+  - bloodyowl
+  - skinnyfoetusboy
```

Comment:
translators:

(we will end up by showing them on the website, an issue is opened about that).